packages/orbit-components/src/Modal/Modal.stories.tsx

```diff
@@ -459,6 +459,44 @@ export const WithItinerary: Story = {
   },
 };
 
+export const WithModalHeaderOnly: Story = {
+  render: args => {
+    const { Container, onClose } = useModal();
+    return (
+      <Container>
+        <Modal onClose={onClose} {...args}>
+          <ModalHeader
+            title="Enjoy something to eat while you fly"
+            illustration={<Illustration name="BaggageDrop" size="small" />}
+            description="Select a flight below to see the menu (where available)"
+          />
+        </Modal>
+      </Container>
+    );
+  },
+
+  args: {
+    mobileHeader: true,
+  },
+
+  parameters: {
+    info: "An example of a modal with a fixed footer. Check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: [
+        "size",
```

Comment:
This one doesn't belong in this story anyway 

```suggestion
        "size",
        "fixedFooter",
```

Reply:
Thanks, added!

I also adjusted the `info` field above to be more accurate.